Source/Core/VideoCommon/RenderBase.h

```diff
@@ -53,9 +53,17 @@ struct EfbPokeData
   u32 data;
 };
 
-// TODO: Move these out of here.
```

Comment:
Is this TODO still relevant for `frameCount`?

Reply:
Probably. I think I'd better readd it.